lib/rules/no-useless-constructor.js

```diff
@@ -25,13 +25,20 @@ module.exports = function(context) {
             return false;
         }
 
-
         return body[0].expression.arguments.every(function(arg, index) {
-            return (arg.type === "Identifier" && arg.name === ctorParams[index].name) ||
```

Comment:
It's fine, you can just incorporate it. You should add some tests that show the benefits of my implementation, such as 

```js
class A extends B { constructor(a, b, ...c) { super(...arguments); } }
```